pkg/discovery/repository/kube_cluster.go

```diff
@@ -192,6 +195,67 @@ func (summary *ClusterSummary) computePodToServiceMap() {
 	}
 }
 
+func (summary *ClusterSummary) computeStaticPodToDaemonMap() {
+	glog.V(2).Info("creating mapping for static pods modeled as daemons.")
+	// static pods are only exposed through the API via mirror pods
+	// https://kubernetes.io/docs/reference/glossary/?all=true#term-mirror-pod
+	mirrorPods := util.GetMirrorPods(summary.Pods)
+	// common name prefix is our assumed conventions for showing intent of creating
+	// daemon sets with static pods
+	prefixToNodeNames := util.GetMirrorPodPrefixToNodeNames(mirrorPods)
```

Comment:
Can we combine the two calls to eliminate one round of iteration?

Reply:
I could, but it would be same amount of work and this way it separates functionality for future use.